tket/tests/tkettestsfiles.cmake

```diff
@@ -59,7 +59,9 @@ set(TEST_SOURCES
     ${TKET_TESTS_DIR}/test_PauliGraph.cpp
     ${TKET_TESTS_DIR}/test_Architectures.cpp
     ${TKET_TESTS_DIR}/test_ArchitectureAwareSynthesis.cpp
-    ${TKET_TESTS_DIR}/Placement/test_Monomorphism_Edge_Break.cpp
```

Comment:
I assume these tests are no longer applicable?
We should delete the file if so.

Reply:
Oops, thought I had. Done